v2/book/examples/AOP.Logging.SourceGenerator.html

```
@@ -0,0 +1,59 @@
+
+<h1>RSCG nr 289 :  AOP.Logging.SourceGenerator</h1>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)